src/components/MAPI/clusters/utils.ts

```diff
@@ -1002,3 +1004,92 @@ export enum ClusterStatus {
   UpgradeScheduled = 'UPGRADE_SCHEDULED',
   UpgradeAvailable = 'UPGRADE_AVAILABLE',
 }
+
+export function getClusterLabels(
+  cluster: capiv1beta1.ICluster
+): IClusterLabelMap {
+  return capiv1beta1.getClusterLabels(cluster);
+}
```

Comment:
Would it be possible to change the return type of `capiv1beta1.getClusterLabels()` to `IClusterLabelMap`? So we can prevent redeclaring this function 😅

Reply:
fixed